backends/mlu/kernels/funcs/mlu_funcs.h

```diff
@@ -63,14 +63,16 @@ inline void TensorCopy(const Context& dev_ctx,
     if (blocking) {
       MemCpyH2D(nullptr, dst_ptr, src_ptr, size);
     } else {
-      AsyncMemCpyH2D(nullptr, stream, dst_ptr, src_ptr, size);
+      C_Device_st device{dst_place_.GetDeviceId()};
```

Comment:
1. move this variable before if branch for code simplification
2. add all devices